src/ng-xform/editable-label/editable-label.component.html

```diff
@@ -1,10 +1,10 @@
-<div *ngIf="!editing" id="{{field.key}}-label">
-  <strong>{{ field.label }}</strong>
+<div *ngIf="!isEditing" id="{{field.key}}-label">
+  <strong>{{ field.label }}</strong> <small *ngIf="isOptional"  style="color: gray">(optional)</small>
```

Comment:
Shouldn't the 'optional' part be only in the editable div? It will never be used here.

Reply:
you're right